app/workers/commit_monitor_handlers/batch/github_pr_commenter/diff_content_checker.rb

```diff
@@ -0,0 +1,48 @@
+module CommitMonitorHandlers::Batch
+  class GithubPrCommenter::DiffContentChecker
+    include Sidekiq::Worker
+    sidekiq_options :queue => :miq_bot_glacial
+
+    include BatchEntryWorkerMixin
+    include BranchWorkerMixin
+
+    def perform(batch_entry_id, branch_id, _new_commits)
+      return unless find_batch_entry(batch_entry_id)
+      return unless find_branch(branch_id, :pr)
```

Comment:
We might want to consider enable/disable for this feature by repo...not sure the best way to break that up or if we should worry about it right now.